cloud-run/src/index.js

```diff
@@ -7,14 +7,13 @@ const configureDomains = require('./configure-domains');
 const action = async () => {
   const serviceAccountKey = core.getInput('service-account-key', { required: true });
   const serviceFile = core.getInput('service-definition') || 'cloud-run.yaml';
-  const regoFile = core.getInput('rego-policy') || 'policy.rego';
```

Comment:
Should this also be removed from action.yml?

Reply:
Yes the readme is still correct around opa flag, there just needs to be a few changes inbefore DAS integration, this PR aims to temporarily disable the configuration parts of opa since the one currently used in the action is no longer necessary moving on to DAS and it would also overwrite the new configurations made by DAS whenever we deploy a new revision. 
I'm uncertain if we want to check for the policy file when deploying with opa enabled as this should be setup upon system creation. We could take a discussion around this action later.